usr/share/rear/backup/NETFS/default/500_make_backup.sh

```diff
@@ -292,7 +292,7 @@ wait $BackupPID
 backup_prog_rc=$?
 
 if [[ $BACKUP_INTEGRITY_CHECK =~ ^[yY1] && "$(basename ${BACKUP_PROG})" = "tar" ]] ; then
-    (cd $(dirname $backuparchive) && md5sum $(basename $backuparchive) > ${backuparchive}.md5 || md5sum $(basename $backuparchive).?? > ${backuparchive}.md5)
+    (cd $(dirname "$backuparchive") && md5sum $(basename "$backuparchive") > "${backuparchive}".md5 || md5sum $(basename "$backuparchive").?? > "${backuparchive}".md5)
```

Comment:
If you quote `$backuparchive`, you should quote `$(dirname "$backuparchive")` and `$(basename "$backuparchive")` as well, otherwise the added quoting does not have much effect.